src/cockpit/389-console/src/lib/replication/replTables.jsx

```diff
@@ -239,20 +239,17 @@ class ManagerTable extends React.Component {
 
     render() {
         return (
-            <div className="ds-margin-top-lg">
-                <Table
-                    className="ds-margin-top"
-                    aria-label="manager table"
-                    cells={this.state.columns}
-                    rows={this.state.rows}
-                    variant={TableVariant.compact}
-                    sortBy={this.state.sortBy}
-                    onSort={this.onSort}
-                >
-                    <TableHeader />
-                    <TableBody />
-                </Table>
-            </div>
+            <Table
```

Comment:
Small thing.
If we have an error then the error has a different font and it's not an italic. I think it's too much different but it's up to you if you want to fix this.
![image](https://user-images.githubusercontent.com/5634892/128041164-abef3b23-d693-4a78-863e-d79ec6c70463.png)

Reply:
Fixed, I also stopped the "refreshing" message that causes the page to constantly bounce during initialization